regression/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ add_subdirectory(goto-diff)
 add_subdirectory(goto-cc-instrument-cbmc)
 add_subdirectory(test-script)
 add_subdirectory(goto-analyzer-taint)
+add_subdirectory(goto-harness)
```

Comment:
:pick: I would prefer that `goto-harness` and `goto-harness-multi-file-project` appeared next to each other. Therefore it may be better to do the re-ordering in the `makefile` instead of the `CMakeLists.txt`.